python/python/glide/async_commands/server_modules/ft_options/ft_aggregate_options.py

```diff
@@ -0,0 +1,307 @@
+    """
+
+    def __init__(self, expression: TEncodable):
+        """
+        Initialize a new FtAggregateFilter instance.
+
+        Args:
+            expression (TEncodable): The expression to filter the results.
+        """
+        self.expression = expression
+
+    def to_args(self) -> List[TEncodable]:
+        """
+        Get the arguments for the Filter clause.
+
+        Returns:
+            List[TEncodable]: A list arguments for the filter clause.
+        """
+        return [FtAggregateKeywords.FILTER, self.expression]
+
+
+class FtAggregateReducer:
+    """
+    A clause for reducing the matching results in each group using a reduction function. The matching results are reduced into a single record.
+    """
+
+    def __init__(
+        self,
+        function: TEncodable,
+        args: List[TEncodable],
+        name: Optional[TEncodable] = None,
+    ):
+        """
+        Initialize a new FtAggregateReducer instance.
+
+        Args:
+            function (TEncodable): The reduction function names for the respective group.
+            args (List[TEncodable]): The list of arguments for the reducer.
+            name (Optional[TEncodable]): User defined property name for the reducer.
+        """
+        self.function = function
+        self.args = args
+        self.name = name
+
+    def to_args(self) -> List[TEncodable]:
+        """
+        Get the arguments for the Reducer.
+
+        Returns:
+            List[TEncodable]: A list of arguments for the reducer.
+        """
+        args: List[TEncodable] = [
+            FtAggregateKeywords.REDUCE,
+            self.function,
+            str(len(self.args)),
+        ] + self.args
+        if self.name:
+            args.extend([FtAggregateKeywords.AS, self.name])
+        return args
+
+
+class FtAggregateGroupBy(FtAggregateClause):
+    """
+    A clause for grouping the results in the pipeline based on one or more properties.
+    """
+
+    def __init__(
+        self, properties: List[TEncodable], reducers: List[FtAggregateReducer]
+    ):
+        """
+        Initialize a new FtAggregateGroupBy instance.
+
+        Args:
+            properties (List[TEncodable]): The list of properties to be used for grouping the results in the pipeline.
+            reducers (List[Reducer]): The list of functions that handles the group entries by performing multiple aggregate operations.
+        """
+        self.properties = properties
+        self.reducers = reducers
+
+    def to_args(self) -> List[TEncodable]:
+        args = [
+            FtAggregateKeywords.GROUPBY,
+            str(len(self.properties)),
+        ] + self.properties
+        if self.reducers:
+            for reducer in self.reducers:
+                args.extend(reducer.to_args())
+        return args
+
+
+class SortOrder(Enum):
```

Comment:
let's replace with an import of OrderBy found in command_args.py
https://github.com/valkey-io/valkey-glide/blob/68199f4870bc50cfec8778eb46d137287e983dda/python/python/glide/async_commands/command_args.py#L32

Reply:
Updated.